how-to/onboarding-guide.md

```diff
@@ -69,8 +107,28 @@ type of situation). Examples of when this might happen include:
 * if there a conflict of interest between a package submitter and the editorial team (e.g. a close colleague of everyone on the team)
 * if the editorial board is at capacity handling the current review load
 
-In this case, you may consider using our internal reviewer signup list to see
-if someone who signed up to be a reviewer might want to serve as an editor.
+In this case, you may consider using our internal reviewer signup list to see if someone who signed up to be a reviewer might want to serve as an editor.
+
+## Process for inviting a first-time community member
+
+### Through the EIC
+
+The Editor in Chief is ultimately the individual responsible for inviting a first-time editor, and when the EIC has identified an editor who is not currently part of the pyOpenSci community, they should:
+
+* Extend a Slack invitation to the individual
+* Welcome the individual in Slack, and provide them with access to the #private-editorial-team and any other channels as needed.
+
+Both the Community Manager and the Executive Director receive a notification whenever someone joins the Slack group, and the Community Manager will ensure that the new member is welcomed and set up with anything they need for Slack.
+
+### Through the Community Manager
+
+Although the EIC is responsible for inviting identified editors to the Slack group, it’s common for the Community Manager to interact with prospective community members that may make good reviewers or editors in the future. As such, at the Community Manager’s discretion they may invite members to Slack, and share the member’s background with the EIC, letting them know if the new community member may make a good reviewer or editor.
+
+In addition, the Community Manager will have the new member fill out the [reviewer form](https://docs.google.com/forms/d/e/1FAIpQLSeVf-L_1-jYeO84OvEE8UemEoCmIiD5ddP_aO8S90vb7srADQ/viewform) or the [editor form](https://docs.google.com/forms/d/17NW0P_h8gpmzf7Fr0cAd8aEbIUWPljPfg4d7Rjs1UIE/edit), as appropriate.
+
+### Through the Executive Director
```

Comment:
this should be the peer review lead. eventually the ED won't do this the review lead will.